iphone/metadata/en-US/release_notes.txt

```diff
@@ -1,53 +1,27 @@
-• Brands are displayed in the Place Page
+We and our contributors have prepared several Christmas/New Year presents for you:
+• Outdoors map style for hiking, cycling, and exploring the Nature
+• Search for addr:place village addresses without streets in Austria, Czechia, Germany, Poland, Slovakia
+• Type "skiing" or "ski run" in the search to find downhill and Nordic pistes
+• OpenStreetMap map data as of December 13
+• Added a button to see photos, reviews, and prices for some hotels. Every booking made through the app contributes to the development of the project!
 
-Routing
-• Don't route through access=emergency, military, agricultural, forestry
-• Tuned ferry crossing estimations
-• Improved cross-country and cross-region routing
-• Improved residential routes
-
-Search
-• Process platz-pl like strasse-str
-• Improved address matching
-• Improved search of brands
-
-Map styling
-• Added flowerbeds, rental shops, bicycle rentals, motorcycle repairs, petting zoo, prison areas, compressed air
-• Added nature reserves and military zones rendering to the car navigation mode
-• Added Docklands Light Railway, Porto metro
-• Display names of ski pistes
-• Tuned/updated icons of fitness centre, yoga, theatre, artwork, statue, theme park, bbq, lighthouse, windmill, heliport & helipad, hunting stand, Kingdom Hall of Jehovah's Witnesses, paid & private versions of parkings and parking entrances, convenience / deli / farm / grocery / health_food shops, tobacco shop
-• Better farmyard & garage landuse area fills
-• Increased scrubs, alpine huts visibility
-• Updated sanitary dump station icons and adjusted its priority
-• Reduced visibility of cave entrances and improved the icon
-• Increased visibility of standalone house numbers
-• Reduced Tokyo subway visibility
-• Removed minor road markers/shields cluttering the map in Estonia
-• Fix colors of Estonia and Germany road markers / shields
-• Increased visibilities of road markers / shields on tertiary and residential road
-• Fixed "gaps" in some densely mapped lines
-• Fixed "gaps" in the outlines of bridges and other compound lines
-
-Outdoors map style preview:
-• Tailored for out-of-the-city outdoor activities like hiking, backcountry cycling/MTB, 4x4, etc.
-• Emphasizes paths/tracks, cliffs, railways, powerlines, rivers and streams, water sources, peaks, mountain saddles and passes, waterfalls, lighthouses, caves, viewpoints, campsites, huts, shelters, picnic sites, gates, stiles, fords, etc.
-• Enable by typing `?olight` in the search field and return to the map; type `?light` to switch back to the main style
-• Use `?odark` and `?dark` for a dark style
-• Developers wanted to add a nice UI for style switching!
+Other notable fixes and improvements:
+• Fixed several crashes and freezing on iOS 12, 13, 14
+• Multiple KML files can be imported from a single KMZ file
+• A credit card is displayed for places with ATM
+• A dollar emoji is displayed where a fee is requred
```

Comment:
should add that its displayed in the POI info sheet / PP only

Reply:
Thanks, will rephrase.